server.js

```diff
@@ -27,7 +33,42 @@ app.get('/', (req, res) => {
   res.send('Hello world')
 })
 
+// Path variable/path parameter
+app.get('/books/:id', (req, res) => {
+  const bookId = req.params.id;
+  const book = booksData.filter(item => item.bookID === +bookId);
```

Comment:
since you are looking for only one book here, there will only be one book with the specific id, you should use the find() method instead of filter(). `find()` returns a single object whereas `filter()` returns an array.